docs/src/modules/javascript/pages/views.adoc

```diff
@@ -242,4 +222,55 @@ Rebuilding a new View may take some time if there are many events that have to b
 
 The View definitions are stored and validated when a new version is deployed. There will be an error message if the changes are not compatible.
 
+[#query]
+== Query reference
+
+Define View queries in a language that is similar to SQL. The following examples illustrate the syntax for a `customers` entity. To retrieve:
```

Comment:
I think we're missing where the table name in the SQL query comes from (from the name of the table you specify when defining your View service)

Reply:
I missed that, good catch!

Reply:
fixed.

Reply:
Update not pushed? Yes, the main point to make for the table name is that it's used to refer to the table in queries.

Reply:
@pvlugter, I'm not sure what happened here. My commits addressing some of your comments apparently wasn't successful. I'm going to try again.

Reply:
See https://github.com/lightbend/akkaserverless-javascript-sdk/pull/139/files line 228.